packages/sdk-effects/iframe-manager/package.json

```diff
@@ -1,6 +1,11 @@
 {
   "name": "@forgerock/iframe-manager",
   "version": "0.0.1",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ForgeRock/ping-javascript-sdk.git",
+    "directory": "packages/sdk-effects/iframe-manager"
```

Comment:
if this is wrong, for example, the beta publish will fail because provenance wont work.